### PR TITLE
Scripts/SQL: AM2 merit bonus + BLU Mag Acc merits

### PR DESCRIPTION
**Pull request by Hozu**
Added magic accuracy and magic burst bonus for AM2 spells
Changed AM2 bonus value from 5 to 1; multiplied the bonus for mag acc and mag burst bonus in the script when needed
Added BLU magic accuracy merit bonus
Copied the recent RDM merit bonus update to applyResistanceEffect for consistency

### CODE REVIEW

**Comment by whasf** [comment]
You updated a lot of whitespace, makes it near impossible to see what you changed ><

**Comment by Hozu** [comment]
Yeah sorry. I was told that if you append ?w=1 to the end of the URL it hides all the whitespace changes.

File: scripts/globals/magic.lua [hunk 1]
```diff
@@ -507,25 +524,20 @@ function applyResistanceEffect(player,spell,target,diff,skill,bonus,effect)
...
-        elseif(element == ELE_WATER) then
-            magicaccbonus = magicaccbonus + player:getMerit(MERIT_WATER_MAGIC_ACCURACY);
-        elseif(element == ELE_WIND) then
-            magicaccbonus = magicaccbonus + player:getMerit(MERIT_WIND_MAGIC_ACCURACY);
-        elseif(element == ELE_ICE) then
-            magicaccbonus = magicaccbonus + player:getMerit(MERIT_ICE_MAGIC_ACCURACY);
-        elseif(element == ELE_LIGHTNING) then
-            magicaccbonus = magicaccbonus + player:getMerit(MERIT_LIGHTNING_MAGIC_ACCURACY);
-        end
+    if (spell:getElement() > 0 and spell:getElement() <= 6 and player:getMainJob() == JOB_RDM and player:getMainLvl() >= 75) then
```
Comment (teschnei) [inline scripts/globals/magic.lua]: you don't need to check job or level

**Comment by Hozu** [comment]
Ahah, whoops. I actually figured that out halfway through making the changes in the first place, but I forgot to remove the check there. Done now.

**Pull request merged**